canonical-web-and-design/ubuntu.com
# Added cloud survey

## Done

- Added cloud survey page at `/cloud/public-cloud-survey` as per [copy doc](https://docs.google.com/document/d/1tKpr0dpZx2K7S2wiBF7QKONNLs2fxz0sFoautszQR1o/edit)

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: https://ubuntu-com-10775.demos.haus/cloud/public-cloud-survey
- Check against copy doc and test all links
- **NOTE: Survey not yet published**


## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/4514

## Screenshots
<img width="1066" alt="Screenshot 2021-11-01 at 16 35 58" src="https://user-images.githubusercontent.com/17607612/139698272-9f4c4037-74e3-4dd8-bfeb-dd8aa613eaa4.png">
